apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,3 +157,11 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
+
+.conversation-view .panel {
```

Comment:
Then I'll go with what I have now, unless we find something better during review.